File: Library/src/app/shared/models/response/rental-response.ts
import {BookResponse} from './book-response';
import {RenteeResponse} from './rentee-response';

export interface RentalResponse {
  id: number;
  status: string;
  startDate: number;
  endDate: number;
  book: BookResponse;
  rentee: RenteeResponse;
}

File: LibraryApi/LibraryApi/DTO/RentalResponse.cs
namespace LibraryApi.DTO
{
    public class RentalResponse
    {
        public int Id { get; set; }
        public string Status { get; set; }
        public long StartDate { get; set; }
        public long? EndDate { get; set; }
        public BookResponse Book { get; set; }
        public RenteeResponse Rentee { get; set; }
    }
}